website/models.py

```diff
@@ -312,6 +313,23 @@ def remove_user(self):
     def get_absolute_url(self):
         return "/issue/" + str(self.id)
 
+    def get_cve_score(self):
+        if self.cve_id is None:
+            return None
+        try:
+            url = "https://services.nvd.nist.gov/rest/json/cves/2.0?cveId=%s" % (self.cve_id)
+            response = requests.get(url).json()
+            results = response["resultsPerPage"]
+            if results != 0:
+                metrics = response["vulnerabilities"][0]["cve"]["metrics"]
+                if metrics:
+                    cvss_metric_v = next(iter(metrics))
+                    score = metrics[cvss_metric_v][0]["cvssData"]["baseScore"]
+                    self.cve_score = score
```

Comment:
My original concern was regarding redundant setting `self.cve_score` here as you also doing it in the views.py L884. Consider just returning it as a result of `Issue::get_cve_score()`. Other option would be rename it to `set_cve_score()`.

Reply:
Done 👍